### PR TITLE
Replace ruby image for non-test env with cwds/ruby:251 and for test with ruby:2.5

### PR DESCRIPTION
**Pull request by money**
https://osi-cwds.atlassian.net/browse/FIT-550

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,24 +1,43 @@
-FROM cwds/ruby:251
+FROM ruby:2.4-alpine
```
Comment (kelfink) [inline Dockerfile]: Did you consider using "FROM cwds/ruby:11" from this project:  https://github.com/ca-cwds/rubyonrails. ?

Comment (money) [reply Dockerfile]: @kelfink The story is asking for a more secure ruby image and after discussion with Steven Grimes, he wants the bare bone version (Alpine) and then add what is necessary.

The link you shared is not using an Alpine version.

**Comment by money** [comment]
Jenkins retest this please

File: Dockerfile (hunk 1 above)
Comment (ratneshraval) [reply Dockerfile]: @money the base ruby image for CWDS should be used and if it is not alpine, can you convert that to alpine and use that in dashboard? That way other repos can use the base image and get the benefit of your hard work :)

**Comment by money** [comment]
Jenkins retest this please

**Comment by edgibbs** [comment]
retest this please

**Comment by money** [comment]
Jenkins retest this please

**Comment by money** [comment]
@kelfink Updated title for clarification

File: docker/test/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM cwds/ruby:251
+FROM ruby:2.5
```
Comment (ratneshraval) [inline docker/test/Dockerfile]: I think this change probably won't work as `ruby:2.5` will not have packages (e.g. node) that `cwds/ruby:251` had

Comment (money) [reply docker/test/Dockerfile]: That is correct. If you look at the file, packages needed to run the test are manually added.

**Pull request merged**